api/codes.py

```diff
@@ -0,0 +1,31 @@
+import json
+import country_converter as coco
+
+cc = coco.CountryConverter()
+
+cities = open('cities.json')
+city_data = json.load(cities)
+
+airports = open('airports.json')
+airport_data = json.load(airports)
+
+
+def convert_country_code(city, country):
+    country_code = cc.convert(names=country, to='ISO2')
+    return convert_city(city, country_code)
+
+
+def convert_city(city, country):
+    city_index = list(filter(lambda x: city_data[x]['name_translations']['en'] ==
+                      city and city_data[x]['country_code'] == country, range(len(city_data))))
+    city_code = city_data[city_index[0]]['code']
```

Comment:
hmm, this piece of code still looks funny to me...maybe add the `[0]` at the end of the list() since we are declaring that as `city_index`, rather than a `city_list`. Also, if possible, extract the `list(filter(lambda))` combo into a utils file, so we can reuse it elsewhere

Reply:
i will make the utils file in the future if I do really need to use it again.